src/app/components/listing/details/criteria/criteria.jsx

```diff
@@ -26,16 +26,26 @@ import {
   qmsStandard,
 } from 'shared/prop-types';
 
-const useStyles = makeStyles({
-  NestedAccordionLevelOne: {
-    borderRadius: '8px',
-    display: 'grid',
-  },
-  NestedAccordionLevelOneSummary: {
-    backgroundColor: '#f9f9f9',
-    borderRadius: '8px',
-  },
-});
+const useStyles = makeStyles(() => ({
```

Comment:
```suggestion
const useStyles = makeStyles({
```

This extra stuff isn't necessary, and I'm deleting it wherever I can. Removing this will mean one of the `)` on line 47 will need to be removed as well.

Also, the indentation of the stuff in this object looks off, but that might go away with the change to the above